features/shop/cart_with_multiple_tax_rates.feature

```diff
@@ -0,0 +1,31 @@
+@ui-cart
+Feature: Apply correct tax for items with different tax rates
+    In order to buy goods with correct taxes applied
```

Comment:
I will try again to make it better `In order to pay proper amount when buying goods from different tax categories`, WDYT?